driver/test/java/io/crate/client/jdbc/integrationtests/MetaDataITest.java

```diff
@@ -69,13 +71,22 @@ public void testGetTables() throws SQLException {
 
     @Test
     public void testGetTablesWithNullSchema() throws SQLException {
-        DatabaseMetaData metaData = conn.getMetaData();
+        PgDatabaseMetaData metaData = (PgDatabaseMetaData) conn.getMetaData();
         ResultSet rs = metaData.getTables("", null, "clus%", null);
 
         assertThat(rs.next(), is(true));
+        assertThat(rs.getString("TABLE_SCHEM"), is("sys"));
         assertThat(rs.getString("TABLE_NAME"), is("cluster"));
         assertThat(rs.getString("TABLE_TYPE"), anyOf(is("SYSTEM TABLE"), is("BASE TABLE")));
 
+        // sys.cluster_health is added to CrateDB 6.0
+        if (metaData.getCrateVersion().compareTo("6.0.0") >= 0) {
+            assertThat(rs.next(), is(true));
+            assertThat(rs.getString("TABLE_SCHEM"), is("sys"));
+            assertThat(rs.getString("TABLE_NAME"), is("cluster_health"));
+            assertThat(rs.getString("TABLE_TYPE"), anyOf(is("SYSTEM TABLE"), is("BASE TABLE")));
+        }
```

Comment:
Do I get it right that we don't need an else branch as next assertions are about sys.cluster?

Is return order of the tables always the same? Can't it be that we hit first cluster and then cluster_health?

Reply:
it used to be 1) `sys.cluster` 2) `test.cluster`, now that we added `sys.cluster_health`, it became 1) `sys.cluster` 1) `sys.cluster_health` 3) `test.cluster`.

The query used under the hood is `ORDER BY TABLE_TYPE,TABLE_SCHEM,TABLE_NAME` so the order should be consistent.